app/controllers/admin/users_controller.rb

```diff
@@ -1,4 +1,25 @@
 module Admin
-  class UserController < BaseController
+  class UsersController < BaseController
+    def change_user_roles
+      binding.pry
+      @user = User.find(params[:id])
+      if params[:roles_names].nil?
+        User::ROLES.each do |role|
+          @user.remove_role(role.to_sym) if @user.has_role?(role.to_sym)
+        end
+      else
+        User::ROLES.each do |role|
+          case
+            when params[:roles_names].include?(role) == false && @user.has_role?(role.to_sym)
```

Comment:
Minor syntax thing: ```case``` and ```when``` are normally in line with each other when indented. IDK why, but that's just the convention I've seen.